pages/research_software/software_releases.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Software Releases
+---
+
+# Software Releases
+
+## Introdution
+
+Developing software is an iterative process which also involves the creation software releases at particular points in time.
+In a software development life cycle this phase is located between testing & quality assurance and maintenance (see diagram below).
+
+![Release phase in software development life cycle](../../assets/img/release-in-sw-dev-life-cycle.png)
```

Comment:
I think the image is nice, but shouldn't it be consistent with the content in the Reference model? We currently have another image.